toolkit/Makefile

```diff
@@ -136,6 +135,12 @@ PACKAGE_URL_LIST   += https://packages.microsoft.com/azurelinux/$(RELEASE_MAJOR_
 REPO_LIST          ?=
 SRPM_URL_LIST      ?= https://packages.microsoft.com/azurelinux/$(RELEASE_MAJOR_ID)/prod/base/srpms
 
+##help:var:PREVIEW_REPO=<path>=Path to the preview repo file to use if 'USE_PREVIEW_REPO=y' is set. If not set, the default preview repo file will be used.
+PREVIEW_REPO                    ?=
+preview_repo_primary_default    ?= $(toolkit_root)/repos/azurelinux-official-preview.repo
+# Intentionally not using $(SPECS) here to avoid issues with custom SPECS directories
+preview_repo_backup_default     ?= $(PROJECT_ROOT)/SPECS/azurelinux-repos/azurelinux-official-preview.repo
+
```

Comment:
This made me think we may also be able to backpaddle with 3.0 a bit and make `PREVIEW_REPO` an "internal" variable. We're allowed to make breaking changes and it's never been documented, which makes it a good candidate and can help us simplify the logic below.

Reply:
Sure, sounds like a plan. Might ruffle a few feathers if people are doing any weird work-arounds but now is the time.